streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -104,7 +104,7 @@
     private final RocksDBMetricsRecorder metricsRecorder;
 
     // visible for testing
```

Comment:
Is this comment still valid?

Reply:
Thanks for the comment.
That part doesn't seem necessary.
And I think you can clear that field.